docs/storage/volume_migration.md

```diff
@@ -201,6 +201,31 @@ spec:
         pod: {}
 ```
 
+## Volume migration cancellation
+
+The users could, for various reasons, wish to stop and cancel the ongoing volume migration.
+Migration cancellations are also handled declaratively, thus users must restore the previous set of volumes, which will be read as a cancellation for the volume update and migration.
+
+A possible way to cancel the migration consists apply the old version of the VM declaration.
```

Comment:
A possible way to cancel the migration is to apply the old version of the VM declaration.